src/DiagnosticMessages.ts

```diff
@@ -17,6 +17,7 @@ export let DiagnosticMessages = {
     callToUnknownFunction: (name: string, scopeName: string) => ({
         message: `Cannot find function with name '${name}' when this file is included in scope '${scopeName}'`,
         code: 1001,
+        functionName: name,
```

Comment:
This feels a little like a hack, but it's either this or I scan the diagnostic message with regex to find the function name.

Reply:
If we want to add a payload to diagnostics we need something more generic.

Reply:
Yeah, good point. I'll implement that.

Reply:
I added a generic `data` property to diagnostic that can contain anything.